Source/Charts/Renderers/YAxisRendererRadarChart.swift

```diff
@@ -139,15 +118,8 @@ open class YAxisRendererRadarChart: YAxisRenderer
         
         if centeringEnabled
         {
-            axis.centeredEntries.reserveCapacity(n)
```

Comment:
This is the same as just initializing a new array and calling reserve capacity. `.map` is optimized to reserve capacity since it knows how many items the result will contain.